packages/react-native-aztec/android/build.gradle

```diff
@@ -1,7 +1,7 @@
 buildscript {
     ext {
         gradlePluginVersion = '4.0.2'
-        kotlinVersion = '1.3.61'
+        kotlinVersion = '1.5.20'
```

Comment:
Previous kotlin version was resulting in some build failures, but unfortunately I can't remember the details of the failure. The upgrade didn't cause any issues, so I don't think we need to be concerned about this version upgrade.